configs/next/packages/pveclib/sources

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Power Vector Library source package and build info
+# ====================================
+#
+
+ATSRC_PACKAGE_NAME="Power Vector Library"
+ATSRC_PACKAGE_VER=1.0.0
```

Comment:
There's no public release of this library, as far as I can see.  Could we ask upstream developers to make one?  Otherwise, we should add something different here.